src/pages/ProfilesExplorerView/domain/events/EventDataReceived.ts

```diff
@@ -0,0 +1,9 @@
+import { BusEventWithPayload, DataFrame } from '@grafana/data';
+
+export interface EventDataReceivedPayload {
```

Comment:
It might be be better being explicit about what was received:

```suggestion
export interface EventSeriesReceivedPayload {
```

Reply:
Good point, I guess it's a tradeoff between a generic event and a specific event.
I'll keep it in mind for the future